script.js

```diff
@@ -1,18 +1,208 @@
+// variables for menu
 const btn = document.querySelector('.hamburguer-btn');
 const menu = document.querySelector('.menu-mobile');
 const li = document.querySelectorAll('.menu-mobile li');
 const logo = document.querySelector('.logo-placeholder');
 const header = document.querySelector('.header');
-
+const main = document.querySelector('.main');
 const listItem = Array.from(li);
-
+// functions for menu
 const menuTgl = () => {
   menu.classList.toggle('visibility');
   logo.classList.toggle('visibility');
   btn.classList.toggle('active');
   header.classList.toggle('active');
 };
-
 btn.addEventListener('click', menuTgl);
+listItem.forEach((child) => { child.addEventListener('click', menuTgl); });
+
+// this is the data for the cards which later on i will start changing in order to make
+// them dinamically
+
+const dataProjects = [
+  {
+    name: 'multi-work series',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/work-img.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
+    name: 'Profesional Art Printing Data 1',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/card-bg.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
+    name: 'Profesional Art Printing Data 2',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/card-bg.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
+    name: 'Profesional Art Printing Data 3',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/card-bg.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
+    name: 'Profesional Art Printing Data 4',
+    info: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tec: ['bootstrap', 'css', 'html', 'ruby'],
+    img: 'img/card-bg.png',
+    seelive: 'link-to-other-ptoject',
+    linkCode: 'link-to-the-repository',
+  },
+  {
```

Comment:
- [Optional] Please try making your popup content look like that of the Figma design, in spacing, and width. For instance, the width of the project image in your popup is small and that of the Figma design is large, with spacing between the title and technology use, and button shape. Please try to follow the Figma design on this.

##### Your design 
![Screenshot from 2022-06-15 18-45-50](https://user-images.githubusercontent.com/58771507/173892620-91b9b9c4-fa62-4d03-8d31-3ea7dabf9f06.png)

##### Figma design
![Screenshot from 2022-06-15 18-46-23](https://user-images.githubusercontent.com/58771507/173892722-44025b61-8910-49b9-8df7-da76d671f301.png)